libsignal-service/Cargo.toml

```diff
@@ -43,3 +43,5 @@ anyhow = "1.0"
 
 [features]
 prefer-e164 = []
+actix = []
+hyper = []
```

Comment:
Did we already want to handle this here?

Reply:
You're right, this is a remnant from... the future. I'll probably open a MR to use features right after this one is merged, but only if it makes sense after I'm done experimenting.